app/js/lib/foxbox/foxbox.js

```diff
@@ -67,6 +67,8 @@ export default class Foxbox extends Service {
     this[p.settings] = new Settings();
     this[p.db] = new Db();
     this[p.net] = new Network(this[p.settings]);
+
+    this.boxes = [];
```

Comment:
I'm OK with adding this change to this patch.
Also it's a good practice to always set the properties of a class to their default values in the `constructor()`. I wonder if eslint can help us with enforcing it as a coding standard.
BAD:
```js
class MyClass {
 constructor() {
 }

 method() {
  this.prop = value;
 }
}
```
GOOD:

```js
class MyClass {
 constructor() {
  this.prop = defaultValue;
 }

 method() {
  this.prop = value;
 }
}
```

What do you think?